1-basics.md

```diff
@@ -14,44 +14,66 @@ kernelspec:
+- [VS Code terminal](https://code.visualstudio.com/docs/terminal/advanced) (xtermjs)
 - [Navigating without a mouse](https://www.youtube.com/watch?v=dJWJ0hCAkAI)
 
-**Exercise**: clone a repo from GitHub and open in Codespaces (or your preferred platform). We will be working with it for the rest of the tutorial. E.g.: numpy (Python + C extensions, could be interesting for debugging), matplotlib, scipy.
+> **Exercise**: Explore VS Code acessability
+>
+> Find a repo on GitHub and try opening it either in your browser (press `.` when on repo page). Try navigating around the UI, and turn on some acessability settings and give them a go!
+
+## Customize your UI
 
-**Setting up your Python project** - We will set up an existing Python project and install the necessary extensions. This piece is fairy quick and self-guided.
+More fun and helpful UI features and settings.
 
-- Extensions marketplace
-- Find and install Python extension
-- Environment management
+- Some key settings to check:
+  - [Customizing Telemetry](https://code.visualstudio.com/docs/getstarted/telemetry)
+  - Color themes: Dark, Colorblind
+  - [Auto save](https://code.visualstudio.com/docs/editor/codebasics#_save-auto-save)
+  - [Files default language](https://code.visualstudio.com/docs/languages/overview)
+- [Settings sync](https://code.visualstudio.com/docs/editor/settings-sync)
+- [Profiles](https://code.visualstudio.com/docs/editor/profiles)
+  - [Python Dev profile template](https://code.visualstudio.com/docs/editor/profiles#_python-profile-template)
+  - [Data Science profile template](https://code.visualstudio.com/docs/editor/profiles#_data-science-profile-template)
+  - [Sarah's profile collection](https://dev.to/crazy4pi314/profiles-for-fun-and-profit-how-to-use-profiles-to-customize-vs-code-57hj)
+- [Extensions marketplace](https://code.visualstudio.com/docs/editor/extension-marketplace)
+  - Don't see what you need? [Make your own!](https://code.visualstudio.com/api/get-started/your-first-extension)
 
-**Exercise**: Create a new virtual environment, install a package with the env create command pallet or the built-in terminal. Then create a new Jupyter notebook or script and import the package.
+> **Exercise**: Share your profile
+>
+> Either in your local or browser-based VS Code, try customizing some settings, themes, etc. either from a blank profile or one of the templates. Then export your current editor config as a gist (or locally if you prefer). If you have the URL for your profile gist, add it to the [discussion page](https://github.com/crazy4pi314/scipy-vscode-tutorial/discussions/17).
 
-- Python extension settings: formatters / linters
-  - available as extensions: pylint, flake8, black, autopep8, ...
+## Setting up for Python development
 
-**Customize your ride** - setting up our environment for Python development
+We will set up an existing Python project and install the necessary extensions. This piece is fairy quick and self-guided.
```

Comment:
so, are we using an existing project or are we creating a new one (as suggested below?)

Reply:
I think for this part, we should so folks can try their hand at the environment management? That dosen't preclued using pre-made ones later I think. Mostly I want to use the workshop devcontainer to control this experience.